Framework/Router.php

```diff
@@ -364,13 +364,8 @@ private function createController($urlInfo, Request $request) {
      *        	Thrown exception
      */
     private function manageError(Throwable $exception, $type = '') {
-        $sendToSentry = true;
 
-        if ($exception instanceof PfmAuthException) {
-            $sendToSentry = false;
-        }
-
-        if($sendToSentry && Configuration::get('sentry_dsn', '')) {
+        if($exception->sendReports && Configuration::get('sentry_dsn', '')) {
```

Comment:
nope, depending on exception, sendReports will not exists (php, db, etc. exception)

Reply:
sendReports exists by inheritance (as property of PfmException) and is overridden when specified in child class. No ?

Reply:
for pfmexceptions yes, not for other exceptions

Reply:
ah, yeah of course